
Extraversion in arch/um/Makefile is not needed in mainline, but just for
separate patches; also, they should set it in the main Makefile, not elsewhere
(Jeff Garzik has just complained).

Signed-off-by: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>
---

 linux-2.6.9-current-paolo/arch/um/Makefile |    9 ---------
 1 files changed, 9 deletions(-)

diff -puN arch/um/Makefile~uml-no-extraversion-in-mainline arch/um/Makefile
--- linux-2.6.9-current/arch/um/Makefile~uml-no-extraversion-in-mainline	2004-09-19 18:27:36.037560456 +0200
+++ linux-2.6.9-current-paolo/arch/um/Makefile	2004-09-19 18:27:36.039560152 +0200
@@ -8,13 +8,6 @@ OS := $(shell uname -s)
 #We require it or things break.
 SHELL := /bin/bash
 
-# Recalculate MODLIB to reflect the EXTRAVERSION changes (via KERNELRELEASE)
-# The way the toplevel Makefile is written EXTRAVERSION is not supposed
-# to be changed outside the toplevel Makefile, but recalculating MODLIB is
-# a sufficient workaround until we no longer need architecture dependent
-# EXTRAVERSION...
-MODLIB := $(INSTALL_MOD_PATH)/lib/modules/$(KERNELRELEASE)
-
 core-y			+= $(ARCH_DIR)/kernel/		 \
 			   $(ARCH_DIR)/drivers/          \
 			   $(ARCH_DIR)/sys-$(SUBARCH)/
@@ -46,8 +39,6 @@ endif
 include $(ARCH_DIR)/Makefile-$(SUBARCH)
 include $(ARCH_DIR)/Makefile-os-$(OS)
 
-EXTRAVERSION := $(EXTRAVERSION)-1um
-
 ARCH_INCLUDE = -I$(ARCH_DIR)/include
 
 # -Derrno=kernel_errno - This turns all kernel references to errno into
_
